src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/FileNamer.kt

```diff
@@ -48,17 +48,18 @@ class FileNamer(
 
     internal fun formatChapterNumber(): String {
         val chapterFormat = if (chapterCount > 99) "%03d" else "%02d"
-        return chapterFormat.format(chapterTitle.toIntOrNull() ?: chapterSort)
+        val chapterNum = chapterFormat.format(chapterTitle.toIntOrNull() ?: chapterSort)
+        return "c$chapterNum"
     }
```

Comment:
Probably not worth a commit, but if you're still working on this PR, I _think_ these can be combined:
```
val chapterFormat = if (chapterCount > 99) "c%03d" else "c%02d"
return chapterFormat.format(chapterTitle.toIntOrNull() ?: chapterSort)
```